.github/workflows/ci_dockers.yml

```diff
@@ -14,8 +14,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python_version: [3.7, 3.8]
-        pytorch_version: [1.5]
+        python_version: [3.6, 3.7, 3.8]
```

Comment:
not sure if weed to test all but with base-cuda it shall be fast and without any extra installation, right?

Reply:
Yes, no extra installation

Reply:
but still it occupy workers which we have limited so lets lower a bit the load...
mind check if we can cache docker images with GH action? could be in another PR...

Reply:
Okay, will check it.

Reply:
Shall we cache base-cuda or conda? or both?

Reply:
ideally both, but let's move it to separate PR and merge this as it is

Reply:
Yes, it will be in a separate PR, this is just asking in advance.